code/modules/mob/living/carbon/human/emote.dm

```diff
@@ -234,7 +234,7 @@
 			if(M_ELVIS in mutations)
 				src.emote("fart")
 				msg = "<B>[src]</B> has left the building..."
-			if(M_HARDCORE in mutations)
+			if(!miming && M_HARDCORE in mutations)
```

Comment:
Still messed up, wrap the in in parentheses.

The reason this works now is because you're doing `(!miming && M_HARDCORE) in mutations`. Because `&&` returns the last value evaluated, if `!miming` is 0 it'll try to do 0 in mutations which fails, if it's true it'll do `M_HARDCORE` in mutations which passes.